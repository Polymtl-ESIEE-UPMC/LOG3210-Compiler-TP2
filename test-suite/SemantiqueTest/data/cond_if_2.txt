Declaration
num a;
EndDeclaration

//good condition (if)
if (true) {
  a = 1 ;
}

  