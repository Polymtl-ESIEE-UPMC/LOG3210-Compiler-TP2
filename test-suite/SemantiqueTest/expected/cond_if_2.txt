{VAR:1, WHILE:0, IF:1, OP:0}